devicon.json

```
@@ -8059,6 +8059,29 @@
         "color": "#383838",
         "aliases": []
     },
+    {
+        "name": "solidjs",
+        "altnames": [],
+        "tags": [
+            "javascript",
+            "framework",
+            "frontend"
+        ],
+        "versions": {
+          "svg": [
+              "original",
+              "original-wordmark",
+              "plain",
+              "plain-wordmark"
+          ],
+          "font": [
+              "plain",
+              "plain-wordmark"
+          ]
+        },
+        "color": "#518ac8",
```

Accord to https://www.solidjs.com/media the primary color should be `#2c4f7c`
```suggestion
        "color": "#2c4f7c",
```